doc/api/events.markdown

```diff
@@ -295,6 +295,24 @@ they were registered, passing the supplied arguments to each.
 
 Returns `true` if event had listeners, `false` otherwise.
 
+### emitter.eventNames()
```

Comment:
Discussed above. Two reasons:
 1. This is less likely to cause conflicts with modules extending `EventEmitter`.
 2. This is much more reasonable name because it correctly described what this method actually does — it returns the list of event names for the registered event listeners. `_events`, for an example, is an object that stores all listeners (grouped by event name).